pkgs/development/interpreters/supercollider/default.nix

```diff
@@ -1,20 +1,20 @@
 { stdenv, fetchurl, cmake, pkgconfig, alsaLib
 , libjack2, libsndfile, fftw, curl, gcc
-, libXt, qtbase, qttools, qtwebkit, readline
-, useSCEL ? false, emacs
+, libXt, qtbase, qttools, qtwebkit, qtwebengine
```

Comment:
We probably don't need `qtwebkit` anymore. The `qtwebengine` replaces it.

Reply:
removed it, thanks!